### DISCUSSION TITLE
DNS problem

### DISCUSSION THREAD

**Discussion by flubber0429** [globaleaks/globaleaks-whistleblowing-software#3851] [Q&A] [answered]
### What version of GlobaLeaks are you using?

4.13.18

### What browser(s) are you seeing the problem on?

Chrome

### What operating system(s) are you seeing the problem on?

Windows, Linux

### Describe the issue

I installed the platform on a debian 10.2 virtual machine on contabo. The installation was successful and the platform is visible using the public IP. However:
1) I created an A record on DNS, but when I try to log in I get a blank page and a 400 error
2) if I try to type the URL instead of the IP in network->https-> hostname, I get unexpected server error
3) I then tried to insert the URL as a new site in sites and enable it, but it gives the same error
I don't see any other configurations to try. There are no firewalls in operation

**Comment by evilaliv3** [comment, marked as answer]
Hello, if you receive error 400 it means you are reaching to the software
with an hostname that is not the one configured on globaleaks.

Ensure to reach the platform with the hostname that you have configured.

You can th configured hostname either check it accessing via Tor, or by
issuing the following command:
gl-admin getvar hostname

If you want you can also reset your hostname by issuing:
gl-admin setvar hostname ""

P.s.: Debian 10 is now very outdated. For a new setup you should use Debian
12

**Comment by flubber0429** [comment]
Ho visto che sei italiano quindi preferisco rispondere in italiano. Grazie intanto per la velocità in un periodo festivo. In fase di installazione non mi ha chiesto un nome per l'hostname. Pensavo lasciasse inserirlo da pannello globaleaks, invece non va. Se provo a inserire altri siti (con relativo dns) non va comunque. C'è un refuso, la debian è in effetti la 12.

**Comment by evilaliv3** [comment]
In fase di completamento del wizard accedevi via IP e credo che possa essere stato conigurato quell'IP come hostname.

Prova ad accedere dunque con l'ip che hai usato durante il wizard o a resettare la configurazione come it ho indicato.

**Reply by flubber0429** [reply]
Grande!!! Ha funzionato. Se aggiungo un ulteriore sito però dà errore 400 su quello. In rete qualcuno dice che va abilitato il multisite, ma non ho trovato questa opzione. Un'ultima cosa poi non disturbo più...la piattaforma funziona sotto ip nattato? In questo caso ho un ip pubblico diretto, ma devo poi installarlo su rete interna

**Reply by evilaliv3** [reply]
Il multisite ora è abilitato di defaul. Una volta non lo era.
Se abiliti un nuovo sito, per raggiungerlo devi accedervi tramite il pannello di gestione della tenant primaria e configurarvi un hostname.

In merito al NAT si funziona assolutamente; vado a rimuovere la possiiblità di configurare come hostname un IP che è ciò che ha causato la problematica.

**Reply by flubber0429** [reply]
Aggiungendo un sito dà errore 400. Però se entro nella configurazione non ha hostname e non so se sia corretto. D'altra parte non mi permette di inserirlo
